### ISSUE TITLE
Not sure it's a issue or not. How could I parse std::string by using CNPY?

### ISSUE THREAD

**Issue by shuming1008** [rogersce/cnpy#75]
Hi Guys, thanks for providing such a convenient tool to parse NPZ.

Now I could parse `double` and `uint64_t` data from NPZ, but I can not parse `numpy.str_` (e.g. `std::string`?)

Is it possible to parse `string` by using CNPY?
If it is YES, could you give me a simple example to parse `string`?

Thanks!

**Comment by shuming1008** [comment]
Additional Note:

I tried to parse `string` by use `char`. Due to `string` is various length.
But it seems the read NPY file is incomplete, if parse NPY file with `string/char` type. I think there must be some issues here.

**Comment by Riley16** [comment]
Yeah, I also am having difficulties loading in string arrays. The first few bytes read in from an array saved as "<UX" dtype are typically correct but other bytes seem to be overwritten with garbage.
